### PR TITLE
[AddIns] Disable deployment/deployment.linux/cbinding by default.

### PR DESCRIPTION
**Pull request by Therzok**
@slluis do we want ifdef'd code on the EnabledByDefault value?

```csharp
EnabledByDefault =
#if MAC || WIN32
    false
#else
    true
#endif
```

?

### CODE REVIEW

**Comment by slluis** [comment]
Yep, let's do it conditionally.

**Comment by Therzok** [comment]
Updated.

File: main/src/core/MonoDevelop.Ide/ExtensionModel/Commands.addin.xml [hunk 1]
```diff
@@ -340,7 +340,7 @@
 	<Command id = "MonoDevelop.Ide.Commands.ProjectCommands.ExportSolution"
 			defaultHandler = "MonoDevelop.Ide.Commands.ExportSolutionHandler"
 			_description = "Convert selected solution to another format"
-			_label = "_Export..." />
+			_label = "Convert solution format..." />
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/ExtensionModel/Commands.addin.xml]: Let's keep this command as Export

Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/ExtensionModel/Commands.addin.xml]: The document wanted this command gone. I thought renaming it to be more explicit was the way to go.

Cc @kallisto

**Pull request merged**